test/integration/basic_test.go

```diff
@@ -181,11 +179,19 @@ func FeatureContext(s *godog.Suite) {
 }
 
 func TestMain(m *testing.M) {
+	flag.StringVar(&givenArgs, "minishift-args", "", "Arguments to pass to minishift")
+	flag.StringVar(&givenPath, "binary", "", "Path to minishift binary")
+	flag.StringVar(&givenTags, "tags", "", "Tags of tests to pass to godog")
+	flag.Parse()
+
+	givenTags = strings.Replace(givenTags, " ", ", ", -1)
```

Comment:
Again, should this not be a bit more defensive? What if no tag is given? Will it run all or nothing?